### ISSUE TITLE
`connections.transform` option conflicts with `no-template-curly-in-string`

### ISSUE THREAD

**Issue by karlhorky** [ts-safeql/safeql#71]
**Describe the bug**

When linting `.eslintrc.js` or `.eslintrc.cjs`, the [built in `no-template-curly-in-string` rule](https://eslint.org/docs/latest/rules/no-template-curly-in-string) reports a problem with [the `connections.transform` syntax](https://github.com/ts-safeql/safeql/pull/19) with a dollar sign in a non-template string:

```js
module.exports = {
  plugins: ['@ts-safeql/eslint-plugin'],
  rules: {
    '@ts-safeql/check-sql': [
      'error',
      {
        connections: [
          {
            databaseUrl: `postgres://postgres:postgres@localhost:5432/postgres`,
            tagName: 'sql',
            transform: '${type}[]', // 💥 Unexpected template string expression. (`no-template-curly-in-string`)
          },
        ],
      },
    ],
  },
};
```

**To Reproduce**

1. Enable the `no-template-curly-in-string` to be a warning or error
2. Lint a `.eslintrc.js` file with the `connections.transform` option as specified in the code block above
3. Observe the problem reported by ESLint

**Expected behavior**

The recommended `connections.transform` configuration shouldn't cause an error with a built-in ESLint rule.

Maybe an alternative syntax that doesn't use the dollar?

**Screenshots**

--

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.6 (21G115)
 - PostgreSQL version 14.2
 - Version `@ts-safeql/eslint-plugin@0.0.14`

**Additional context**

--

**Comment by karlhorky** [comment]
Yeah I've seen it a few times.

> What would you recommend using instead?

Maybe just the curly brackets without the dollar sign? Or if that's too ambiguous, could switch the curlies to parentheses or angle brackets (either with or without dollar)

**Comment by karlhorky** [comment]
Thanks for the fix in `@ts-safeql/eslint-plugin@0.0.16` @Newbie012! 🙌  We'll try it out soon.
